calculator/prompt.c

```diff
@@ -17,7 +17,7 @@ int prompt() {
       printf("bye bye!\n");
       return 0;
     }
-    printf("calc > %s\n", buffer);
+    printf("calc > %s\n", compute(buffer));
```

Comment:
You can see here that I can call the `compute` parameter because it is a function pointer, also notice that we pass in a `char *` and the function returns a `char *` which we use to print out to `stdout` (the screen/terminal).